template/config/playwright/playwright.config.ts

```diff
@@ -98,7 +98,11 @@ const config: PlaywrightTestConfig = {
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: 'npm run dev',
+    /**
+     * Use the dev server by default for faster feedback loop.
+     * Use the preview server on CI for more realistic testing.
+     */
```

Comment:
```suggestion
    Playwright will re-use the local server if there is already a dev-server running.
     */
```

something like that is missing, then its excellent!